src/cli/config.rs

```diff
@@ -36,6 +36,8 @@ pub struct Config {
     pub daemon_user: String,
     #[serde(default)]
     pub daemon_group: String,
+    #[serde(default)]
+    pub collection_interval: u64,
```

Comment:
The collection_interval has no default value.

Reply:
Default value added through `fn default_collection_interval() -> u64`.

Reply:
Default value added through `fn default_collection_interval() -> u64`.